docs/controller-skills/military.md

```diff
@@ -69,10 +71,33 @@ These Restricted Areas may be activated in vatSys, and the controllers of these
 Unless otherwise stated in Local Instructions, Restricted Areas activated for control by Military TCUs and Towers, will become **Class C** airspace.
 
 ## Formation Flights
-Reserved.
+A standard military formation consits of a proximity of no more than 1 mile laterally or longitudinally and within 100 feet vertically from the flight leader is maintained by each wingman. As such you treat both aircraft as one and ensure only the leader is on mode C. Futhermore per FIHA, Military AIrcraft are given discrete sqwaks in the 5000's and 6000's.
```

Comment:
Could you please define what FIHA is, either on the abbreviations.md page, or within this page? Or even an embedded link to the page would work.

Also, small typo with the capitalised I in AIrcraft, as well as "squawk".

This line would need a bit of fleshing out. Military aircraft aren't quite assigned discrete codes in the 5000s and 6000s. When they're operating in civil airspace, they get normal codes like everyone else. But obviously military flights not receiving a service and needing to squawk a non-discrete code, will squawk 6000 in class G, 5000 in all other classes of airspace.

Reply:
FIHA: Flight Info Handbook Australia